lib/webpack-config/__snapshots__/create-webpack-config.test.js.snap

```diff
@@ -83,7 +83,6 @@ Object {
         ],
       },
     ],
-    "strictExportPresence": true,
```

Comment:
I couldn't see any mention of this property in Webpack 4 docs, better if we not use it.